x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/dataframe/DataFrameAnalyticsManager.java

```diff
@@ -304,6 +326,8 @@ private void reindexDataframeAndStartAnalysis(DataFrameAnalyticsTask task, DataF
         // Create destination index if it does not exist
         ActionListener<GetIndexResponse> destIndexListener = ActionListener.wrap(
             indexResponse -> {
+                ImmutableOpenMap<String, MappingMetadata> mappings = indexResponse.mappings();
+                Map<String, Object> sourceAsMap = mappings.valuesIt().next().getSourceAsMap();
```

Comment:
```suggestion
```
Seems unused?

Reply:
Ah, leftover I forgot to delete. Thanks for catching!